src/contracts/Arbitrator.ts

```diff
@@ -54,4 +56,10 @@ export class Arbitrator {
       this.logger.throwError(e.message);
     }
   };
+
+  givingRulingCall = async(disuputeId:number, ruling:number): Promise<any> => {
```

Comment:
Update the function name to `giveRulingCall`

Reply:
Yeah, @devpavan04 is finishing it up